android/app/src/main/res/layout/item_sponsor.xml

```diff
@@ -18,6 +18,17 @@
             android:layout_margin="@dimen/layout_margin_small"
             android:contentDescription="@string/sponsor_image" />
 
+
+        <TextView
+            android:id="@+id/sponsor_name"
+            android:layout_width="79dp"
```

Comment:
Why a fixed size?
I think this is the reason as to why the Sponsor name isn't properly centralized for most of the sponsors.